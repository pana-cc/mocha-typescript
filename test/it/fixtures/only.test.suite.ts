import { only, pending, skip, slow, suite, test, timeout } from "../../../index";

@suite class Suite1 {
    @test public test1() {}
    @test.only public test2() {}
}
