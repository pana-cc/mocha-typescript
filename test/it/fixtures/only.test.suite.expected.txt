
  Suite1
    âœ“ test2

  1 passing
